src/bin/nydus-image/core/bootstrap.rs

```diff
@@ -610,7 +623,28 @@ impl Bootstrap {
             Result<()>
         )?;
 
-        // Erofs does not have inode table, so we lose the chance to decide if this
+        // `Node` offset might be updated during above inodes dumping. So `get_prefetch_table` after it.
+        let prefetch_table = ctx
+            .prefetch
+            .get_rafsv6_prefetch_table(&bootstrap_ctx.nodes, meta_addr);
+
+        if let Some(mut pt) = prefetch_table {
+            // Device slots are very close to extended super block.
+            ext_sb.set_prefetch_table_offset(prefetch_table_offset);
+            ext_sb.set_prefetch_table_size(prefetch_table_size);
+            bootstrap_writer
+                .seek_offset(prefetch_table_offset as u64)
+                .context("failed seek prefetch table offset")?;
+
+            pt.store(&mut bootstrap_writer).unwrap();
+
+            bootstrap_writer.file.seek(SeekFrom::Start(ext_sb_offset))?;
+            ext_sb
+                .store(&mut bootstrap_writer)
+                .context("failed to revise extended SB")?;
```

Comment:
why ext_sb has to be written multiple times?

Reply:
Because, at the very beginning, I didn't want to touch current code too much.
So it updates extended block when it has to since some fields/info of it change. If the first `store` didn't happen, inodes region can't relocate to its offset from bootstrap. As the re-writing won't bring too much latency or overhand, it is somewhat like a `seek`.

Reply:
Looks like the ext_sb can be seeked to via `ext_sb_offset` and be written once we finishes setting `ext_sb`. Does it not work somehow?

> If the first store didn't happen, inodes region can't relocate to its offset from bootstrap. 

Does the inode region depend on ext_sb location? I don't quite get it as ext_sb is fixed sized. We can always seek over it.

Reply:
Done.